ui/testem.js

```diff
@@ -10,7 +10,8 @@ const config = {
         // --no-sandbox is needed when running Chrome inside a container
         process.env.CI ? '--no-sandbox' : null,
         '--headless',
-        '--disable-gpu',
+        // as per https://github.com/ember-cli/ember-cli/pull/8774
```

Comment:
We don't need to comment out the line here.  You can just include a link to the ember-cli pr in the PR description. Go ahead and remove the `'--disable-gpu--'`

Reply:
Okies, removed that line, and updated the original PR comment with that ember-cli link.

Reply:
Can you also remove the comment (sorry, I didn't make that clear).  With the ember-cli link now in the PR comment, no need to have it in the code.  Thanks!

Reply:
That's sort of what I thought, but didn't quite get - update coming.

Reply:
done.